From: Robert Schwebel <r.schwebel@pengutronix.de>
Subject: [patch] remove -Werror

With gcc-4.3.2 I get this error:

cc1: warnings being treated as errors
lexer.c:1638: error: 'input' defined but not used
make[3]: *** [lexer.lo] Error 1

Fix it temporarily by removing -Werror.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 src/Makefile.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: confuse-2.6/src/Makefile.am
===================================================================
--- confuse-2.6.orig/src/Makefile.am
+++ confuse-2.6/src/Makefile.am
@@ -10,7 +10,7 @@ localedir = $(datadir)/locale
 DEFS = -DLOCALEDIR=\"$(localedir)\" @DEFS@
 
 if cc_is_gcc
-AM_CFLAGS=-Wall -Werror
+AM_CFLAGS=-Wall
 endif
 
 LIBS = @LIBS@
